docs/release-notes/NuGet-5.9.md

```diff
@@ -0,0 +1,250 @@
+---
+title: NuGet 5.9 Release Notes
+description: Release notes for NuGet 5.9 including new features, bug fixes, and DCRs.
+author: erdembayar
+ms.author: eryondon
+ms.date: 3/11/2021
+ms.topic: conceptual
+---
+
+# NuGet 5.9 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.9**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.9](https://visualstudio.microsoft.com/downloads/) | [5.0](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
+  
+> [!NOTE]
+> Visual Studio 16.9, MSBuild 16.9, and .NET 5.0 require NuGet.exe 5.9 or later.
+
+## Summary: What's New in 5.9
+
+* Show requested version in the Solution Level UI - [#9827](https://github.com/NuGet/Home/issues/9827)
+
+* Extend the `.nupkg.metadata` file to include the installation source - [#10354](https://github.com/NuGet/Home/issues/10354)
+
+* Add Update context menu item for package dependencies that launches Package Manager UI with preselected packages to update - [#10378](https://github.com/NuGet/Home/issues/10378)
+
+* Enable recommender for PackageReference style projects - [#10053](https://github.com/NuGet/Home/issues/10053)
+
+* Introduce a new msbuild property to exclude build output for specific tfms during pack task - [#10396](https://github.com/NuGet/Home/issues/10396)
+
+* INuGetPackageFileService - Fetch Images and embedded licenses for Codespaces-connected and standalone - [#10151](https://github.com/NuGet/Home/issues/10151)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* The down icon icon when the latest package version is installed is not intuitive. The old green tick was perfect - [#9789](https://github.com/NuGet/Home/issues/9789)
+
+* Nuget Debug verbosity should say where a package came from - [#3055](https://github.com/NuGet/Home/issues/3055)
+
+* NuGet pack should catch incorrect omitting of the dot in version numbers - [#9215](https://github.com/NuGet/Home/issues/9215)
+
+* [CPVM] Disable pinning of the central transitive dependencies  - [#10132](https://github.com/NuGet/Home/issues/10132)
+
+* net5 tfm: produce error when missing TPV - [#9441](https://github.com/NuGet/Home/issues/9441)
+
+* Log package contenthash during restore logging (during extraction) - [#10384](https://github.com/NuGet/Home/issues/10384)
+
+* Implement a pre-registration mechanism for legacy PR projects that call restore at solution open - [#9986](https://github.com/NuGet/Home/issues/9986)
+
+* NuGet package recommender should work when more than one source is selected in package manager - [#10433](https://github.com/NuGet/Home/issues/10433)
```

Comment:
Remove as not public

Reply:
Leaving as summary mentions the A/B feature & therefore will hint at it in the release blog for whats to come in 5.10.